renderer/components/Directory.jsx

```diff
@@ -1,76 +1,94 @@
-import React from 'react';
+import React, { Component } from 'react';
+import cx from 'classnames';
 import File from './File.jsx';
 import CreateMenu from './CreateMenu.jsx';
 import CreateForm from './CreateForm.jsx';
 import RenameForm from './RenameForm.jsx';
 
-export default class Directory extends React.Component {
-  constructor() {
-    super();
-  }
+export default class Directory extends Component {
+  handleListItemClick = proxyEvent => (
```

Comment:
Arrow functions in classes are really hard to read.  Maybe you should write them in a classical way like:
```
handleListItemClick(proxyEvent) {

    this.props.clickHandler(
        this.props.id,
        this.props.directory.path,
        this.props.directory.type,
        proxyEvent
    );
}
```

That change should also preserve a consistent code style because the handlers in the `App` component were not written as Arrow functions. 

Also, wouldn't the methods `handlePlusIconClick` and `handleListItemClick` need to be bound to the `this` instance in the  Component constructor?